td-models/api.yml

```diff
@@ -296,7 +380,9 @@ components:
                 type: string
               weights:
                 type: array
+                description: denote inferred edge weights from the engines. e.g. for DySE this will be [level, trend], for Delphy this will be [weight]
```

Comment:
```suggestion
                description: denote inferred edge weights from the engines. e.g. for DySE this will be [level, trend], for Delphi this will be [weight]
```